.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,66 @@
+name: Java CI
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Do we want to pin this? Based on the [update to Ubuntu 20](https://github.com/actions/virtual-environments/issues/1816), it looks like they're happy to include updates of installed versions which have breaking changes.

Reply:
I would say that if we become incompatible with new versions of Ubuntu, it's good to be aware of it as soon as possible, so that's why `ubuntu-latest` makes intuitive sense to me.

Reply:
They apparently generate artificial build "warnings" when they're getting ready to change what "latest" points to, so I suppose we can use that as a prompt to proactively test. Having avoidable unscheduled build breakages isn't really the best (as we've found out).